.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: c
```

Comment:
I stripped the original comment from astropy's template.

```
# We set the language to c because python isn't supported on the MacOS X nodes
# on Travis. However, the language ends up being irrelevant anyway, since we
# install Python ourselves using conda.
```